src/vs/workbench/browser/workbench.ts

```diff
@@ -208,6 +214,11 @@ export class Workbench implements IPartService {
 				viewletAndEditorPromises.push(this.sidebarPart.openViewlet(viewletId, false).then(() => viewletTimerEvent.stop()));
 			}
 
+			if (!this.panelHidden) {
+				const panelId = this.storageService.get(PanelPart.activePanelSettingsKey, StorageScope.WORKSPACE);
+				viewletAndEditorPromises.push(this.panelPart.openPanel(panelId, false));
```

Comment:
:heavy_check_mark: 
I use compositeAndEditorPromises, since some editor promises get appended at the end and createCompositesPromises is not got due to this